backend/app/app/settings.py

```diff
@@ -27,6 +27,9 @@
 
 ALLOWED_HOSTS = []
 
+# AUTH_USER_MODEL = 'polls.User'
+LOGIN_REDIRECT_URL = '/'
```

Comment:
Login redirect is used for django templates, we probably don't need it and it should be removed